src/scripts/components/joinForm.js

```diff
@@ -0,0 +1,116 @@
+import { entries as getFormEntries } from '../utilities/formData';
+
+class JoinForm {
```

Comment:
is this something we can use instead of ak-js's version on actionkit, too? (I'm vaguely ok removing ak-js from giraffe, and then consolidating cross-site JS here)
I kinda feel wherever the JS should exist, we should have it in a single place...

Reply:
As long as the classes and key parts of the markup matches, it'll work. I didn't look at what was in ak-js; I just rewrote the old WordPress homepage signup widget JS for giraffe markup/CSS.